### PR TITLE
Added JP

### PR DESCRIPTION
**Pull request by Achu2234**
I tried my best

### CODE REVIEW

**Comment by Achu2234** [comment]
@New-dev0 Sir I tried my best

File: strings/strings/jp.yml [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+help_5: "• アドオン"
+help_6: "••ボイスチャット"
+help_7: "インラインプラグイン••"
+help_8: "⚙️ オーナ ツール
+help_9: "設定 ⚙️"
+help_10: "••Cʟᴏꜱᴇ••"
+help_11: "Plugin Name-{}\n\n✘ Commands Available -\n\n"
+
+
+# userlogs
```
Comment (New-dev0) [inline strings/strings/jp.yml:35]: strings (" ") are missing here, making it invalid yaml file.
Can you please fix it

**Comment by New-dev0** [comment]
Thanks, for your contribution.
Please fix that string issue and there are some other strings too on the queue to be commited in dev branch.

It would be better, if you can include that too.

**Comment by Achu2234** [comment]
@New-dev0  ok sir and can u gib strings too on the queue to be commited in dev branch.

**Comment by New-dev0** [comment]
@Achu2234 https://github.com/TeamUltroid/Ultroid/compare/a0fc9894083b...0b4405c102ab

**Comment by Achu2234** [comment]
ok sir

_

> help_1： "` {} `は有効なプラグインではありません！"
> help_2：「ボットがインラインクエリに応答しませんでした。\ n` {}再起動 `の使用を検討してください。」
> help_3：「 `@Botfatherからボットのインラインモードをオンにしてください。"
> help_4： "•プラグイン"
> help_5： "•アドオン"
> help_6： "••ボイスチャット"
> help_7：「インラインプラグイン••」
> help_8：「⚙️オーナーツール」
> help_9：「設定⚙️」
> help_10： "••Cʟᴏꜱᴇ••"
> help_11： "プラグイン名-{} \ n \n✘使用可能なコマンド-\ n \ n"

_

sir is the string now ok @New-dev0

**Comment by New-dev0** [comment]
> https://github.com/TeamUltroid/Ultroid/pull/228#issuecomment-989881311

No, every string should start and end with "

**Comment by Achu2234** [comment]
@New-dev0 sir can u fix it plx i am now adding new strings

**Comment by New-dev0** [comment]
, 

> @New-dev0 sir can u fix it plx i am now adding new strings

Ok, you can complete adding first, then I will be able to do, else it will create conflict

**Comment by Achu2234** [comment]
@New-dev0 ok sir i will gib new pull

**Comment by Achu2234** [comment]
https://github.com/TeamUltroid/Ultroid/pull/229
@New-dev0

**Comment by New-dev0** [comment]
This can be closed now!

**Pull request closed without merging**